paasta_tools/setup_kubernetes_job.py

```diff
@@ -113,8 +113,19 @@ def main() -> None:
         logging.getLogger("kazoo").setLevel(logging.WARN)
         logging.basicConfig(level=logging.INFO)
 
+    # emit deploy events for updated jobs
     deploy_metrics = metrics_lib.get_metrics_interface("paasta")
 
+    # emit timing metrics for s_k_j
+    cluster = args.cluster or load_system_paasta_config().get_cluster()
+    timer = metrics_lib.system_timer(
+        dimensions=dict(
+            cluster=cluster,
+            eks=args.eks,
+        ),
+    )
+    timer.start()
```

Comment:
comment: i think we could technically do the same dimension update with a context manager - but it might be a bit more painful for not much gain :p